Kunena/Kunena-Forum
# Extending Social Sharing with Custom tag

Pull Request for New Feature

The implementation of jssocials sharing buttons works good, but a lot of websites have their own social sharing plugin.
This PR adds the possibiity to use 'whatever' social sharing plugin your website has and integrate it into Kunena to get a consistent look and feel all over your website.

#### Summary of Changes
Add crypsis(b3) configuration option to add Socials Sharing Custom Tag ( in template / features), added new layout widget

#### Testing Instructions
In Kunena template configuration > select crypsis(b3) > goto Featured tab and configure settings
> Display Social Share (no / yes / Custom tag): Custom tag
> Social Share Custom Tag: {jssocials} (as an example for the ochjssocials plugin)

When visiting a topic on the forum, the new social sharing buttons should appear and be exactly the same as for all other Joomla pages (where the jssocials plugin is used).

This can also be tested with other plugins and even be used for other functionality :)

For the Kunena forum this would fix the issue you see here: https://www.kunena.org/forum/76-Official-Announcements-and-News-about-Kunena/139657-kunena-5-0-6-released
in the blog the {jssocials}  plugin is used with css plain. on the forum (crypsys) the css classic is used.
Now you see a combination of both on the kunena page.
When configuring the crypsis template to use custom tag {jssocials} this issue is fixed AND you are able to configure all {jssocial} settings in one place!

Please see me with any questions!
